src/main.tsx

```diff
@@ -51,7 +51,7 @@ root.render(
                 <Route path={"admin/import"} element={ <CSVUploadPage/>}/>
                 <Route path={"employee/new-issue"} element={<NewIssueEmployee />}/>
                 <Route path={"employee/users"} element={<EmployeeUserList />}/>
-                <Route path="/users/:userId/info" element={<EmployeeUserInformation />} />
```

Comment:
`{"employee/users/:userId"}`

is het echt zo lastig

Reply:
hahaha sorry was moe